modules/navier_stokes/doc/content/source/auxkernels/kOmegaSSTF1BlendingAux.md

```diff
@@ -0,0 +1,30 @@
+# kOmegaSSTF1BlendingAux
+
+Computes the $F_1$ blending function for the $k-\omega$ SST model.
+The function is defined as follows:
+
+\begin{equation}
+F_1 = \operatorname{tanh} \left[ \left[ \operatorname{min} \left( \operatorname{max} \left( \frac{\sqrt{k}}{\beta^* \omega d}, \frac{500 \mu}{\rho d^2 \omega} \right), \frac{2k}{d^2 {CD}_{k \omega}} \right) \right]^4 \right] \,,
+\end{equation}
+
+where:
+
+- $k$ is the turbulent kinetic energy,
+- $\omega$ is the turbulent kinetic energy specific dissipation rate,
+- d$ is the distance to the nearest wall,
+- $\mu$ is the dynamic viscosity,
+- $\rho$ is the density,
+- ${CD}_{k \omega} = \operatorname{max} \left( \frac{\nabla k \cdot \nabla \omega}{\omega}, 10^{-20} \right)$ is the positive part of the cross diffusion term,
+- $\beta^* = 0.09$ is a closure parameter.
+
+To calculate the distance to the nearest wall, we recommend using [WallDistanceAux](WallDistanceAux.md).
```

Comment:
Now the updated F1 blending function automatically computes the distance map if 'walls' is provided. May consider commenting about this feature instead of line 20.